packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -1050,9 +1331,13 @@ function TableCheckboxCell({cell}) {
             },
             classNames(
               stylesOverrides,
-              'react-spectrum-Table-cell'
+              'react-spectrum-Table-cell',
+              {
+                'react-spectrum-Table-checkboxCell--isDraggable': isTableDraggable
```

Comment:
what's this for? i thought it would be so I could drag starting on a checkbox, but I don't see that as doing anything